dynacred/tsconfig.json

```diff
@@ -19,6 +19,7 @@
     "outDir": "./build"
   },
   "include": [
-    "src"
+    "src",
+    "src/group"
```

Comment:
it should work without that - can you try?

Reply:
It is the case